content/en/docs/languages/php/basics.md

```diff
@@ -29,20 +29,25 @@ Use [another language](/docs/languages/) to create a gRPC server.
 
 The example code for our tutorial is in
 [grpc/grpc/examples/php/route_guide](https://github.com/grpc/grpc/tree/{{< param grpc_vers.core >}}/examples/php/route_guide).
-To download the example, clone the `grpc` repository by running the following
+To download the example, clone the `grpc` repository and its submodules by running the following
 command:
 
 ```sh
-$ git clone -b {{< param grpc_vers.core >}} https://github.com/grpc/grpc
+$ git clone --recurse-submodules -b {{< param grpc_vers.core >}} https://github.com/grpc/grpc
 ```
 
-You need grpc-php-plugin to help you generate proto files. You can build it from source:
+You need the `grpc-php-plugin` to help you compile `.proto` files. Build it from source as follows:
 
 ```sh
-$ cd grpc && git submodule update --init && make grpc_php_plugin
+$ cd grpc
+$ mkdir -p cmake/build
+$ pushd cmake/build
+$ cmake ../..
+$ make protoc grpc_php_plugin
```

Comment:
@jtattermusch @stanley-cheung - do we want to include the `protoc` target here too? (This is what is done in https://github.com/grpc/grpc/pull/25544).

Reply:
OK to me.